technical-embedded-ui/tpp-ui/src/app/account-selector/account-selector.component.html

```diff
@@ -1,6 +1,6 @@
 Choose account access
-<mat-radio-group class="account-radio-group" name="accessSelector">
-  <mat-radio-button #dedicated class="account-radio-button" [value]="dedicatedChecked.checked" [checked]="dedicatedChecked.checked" aria-label="Dedicated accounts access"></mat-radio-button>
+<mat-radio-group class="account-radio-group" [formControl]="accessSelector" (click)="clearConsent()">
```

Comment:
We will have to avoid using angular material since we use bootstrap

Reply:
Nope, technical UI uses material